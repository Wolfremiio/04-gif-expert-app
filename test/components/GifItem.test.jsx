import { render } from "@testing-library/react";
import { GifItem } from "../../src/components/GifItem";


render

describe('Testing in <GifItem>', () => {

  const title= "Wolf";
  const url = "http://GifMixApp.com";

  test('Should make match with the snapshot ', () => {
    const {container}= render (<GifItem url={ url } title={ title}/>)

    expect(container).toMatchSnapshot();
  });
})


  