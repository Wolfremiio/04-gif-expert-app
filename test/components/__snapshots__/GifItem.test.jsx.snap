// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing in <GifItem> Should make match with the snapshot  1`] = `
<div>
  <div
    class="card "
  >
    <img
      alt="Wolf"
      src="http://GifMixApp.com"
    />
    <p>
      Wolf
    </p>
  </div>
</div>
`;
